From 1bcce0575285e368c1aeb7198c843b82a3f81fa1 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Wed, 17 Oct 2018 18:12:07 +0800
Subject: [PATCH] arm64: rk1808_linux_defconfig: enable SRAM

Enable the Generic on-chip SRAM driver for VAD and NPU.

Change-Id: Ieb5cbb6ae053ec7a8c8b6ffe0921576a80f17636
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 arch/arm64/configs/rk1808_linux_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/rk1808_linux_defconfig b/arch/arm64/configs/rk1808_linux_defconfig
index bfb5217c8846..df5b88c37356 100644
--- a/arch/arm64/configs/rk1808_linux_defconfig
+++ b/arch/arm64/configs/rk1808_linux_defconfig
@@ -94,6 +94,7 @@ CONFIG_DEVTMPFS=y
 CONFIG_DEVTMPFS_MOUNT=y
 # CONFIG_ALLOW_DEV_COREDUMP is not set
 # CONFIG_BLK_DEV is not set
+CONFIG_SRAM=y
 CONFIG_SCSI=y
 CONFIG_BLK_DEV_SD=y
 CONFIG_CHR_DEV_SG=y
-- 
2.35.3

